ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -3,21 +3,101 @@
 //  Licensed under the MIT License.
 //
 
-import Combine
-import UIKit
 import SwiftUI
+import UIKit
+
+/// View that represents the button
+public struct MSFButtonView: View {
```

Comment:
Yeah. So far the Button is the only problem as far as naming conflicts we've seen.
I thought on suggesting to change the struct name to just "ButtonView" to prevent extra workarounds to fix the clash.
Do you have any suggestions?

Reply:
As discussed offline: we'll continue this conversation separately, since it's a tricky one.